services/agent/src/simcore_service_agent/modules/volumes_cleanup/_docker.py

```diff
@@ -18,10 +18,10 @@ async def get_dyv_volumes(docker: Docker, target_swarm_stack_name: str) -> list[
     dyv_volumes: deque[dict] = deque()
     volumes = await docker.volumes.list()
     for volume in volumes["Volumes"]:
+        volume_labels = volume.get("Labels", {}) or {}
```

Comment:
I think it will be easer it read as (the default ``{}`` tricked me!
```python
volume_labels: dict[str, Any] = volume.get("Labels") or {}
```

should work for all these cases

```python
volume1 = { }
volume2 = {"Labels": None }
volume3 = {"Labels": {} }
volume4 = {"Labels": {"key": value, ... } }
```